frontend/src/routes.tsx

```diff
@@ -8,7 +8,7 @@ export function Routes() {
   return (
     <Layout>
       <Router>
-        <DashboardPage/>
+        <DashboardPage path="/dashboard" />
```

Comment:
Let's add the default prop to this route:

![image](https://user-images.githubusercontent.com/25971028/173489987-dca5e6e9-d26e-4c24-a9fb-165b76c9e9c5.png)

Please, take a look at the documentation for Reach Router: https://reach.tech/router/tutorial/08-default-routes
For